fe/fe-core/src/main/java/org/apache/doris/planner/Planner.java

```diff
@@ -101,9 +101,7 @@ private void setResultExprScale(Analyzer analyzer, ArrayList<Expr> outputExprs)
                             }
 
                     if (slotList.contains(slotDesc.getId()) && null != slotDesc.getColumn()) {
-                        // TODO output scale
-                        // int outputScale = slotDesc.getColumn().getType().getScale();
-                        int outputScale = 10;
+                        int outputScale = slotDesc.getColumn().getScale();
```

Comment:
Are you sure the scale is always >= 0?

Reply:
it is only for decimal, if scale < 0,  be will ignore it